### DISCUSSION TITLE
Set default proxy and API addresses for docker image?

### DISCUSSION THREAD

**Discussion by zwolfinger** [hoppscotch/hoppscotch#4129] [Q&A]
We would like to enable the proxy and set the default proxy address and API address for our users.  Is this possible when using the hoppscotch/hoppscotch AIO docker image?

We used to replace them in newstore/settings.ts and newstore/RESTSession.ts

Is building our own docker image "the way" here?

**Comment by jimmybrancaccio** [comment]
This would definitely be awesome to have a `PROXY_URL` environment variable so every user doesn't have to manually go to the Settings page and set it.
